### PR TITLE
Improve PR wording so maintainers don't entirely gloss over DMAPI updates

### PR DESCRIPTION
**Pull request by Cyberboss**
image

### CODE REVIEW

File: .github/workflows/update_tgs_dmapi.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
         source_branch: "tgs-dmapi-update"
         destination_branch: "master"
         pr_title: "Automatic TGS DMAPI Update"
-        pr_body: "This pull request updates the TGS DMAPI to the latest version. Please note any breaking or unimplemented changes before merging."
+        pr_body: "This pull request updates the TGS DMAPI to the latest version. Please note any changes that may be breaking or unimplemented in your codebase by checking the definitions file: code/__DEFINES/tgs.dm before merging."
```
Comment (optimumtact) [inline .github/workflows/update_tgs_dmapi.yml:44]: check it for what

**Comment by Mothblocks** [comment]
its ok they dont merge my prs either

**Comment by ZephyrTFA** [comment]
You cannot convince me to read dmapi prs

File: .github/workflows/update_tgs_dmapi.yml (hunk 1 above)
Comment (Cyberboss) [reply .github/workflows/update_tgs_dmapi.yml:44]: For changes

**Comment by Cyberboss** [comment]
@ZephyrTFA Literally this is all you had to look at for the latest one
image
Here's an example of a bigger one that actually required looking at.
image

**Pull request merged**